Gemfile

```diff
@@ -74,4 +74,8 @@ group :test do
 
 end
 
-gem 'rails-i18n'
\ No newline at end of file
+gem 'rails-i18n'
+
+gem 'pry'
```

Comment:
This gem is used for debugging in development and test environments, my suggestion is to move it to the dev & test group.

```
group :development, :test do
   gem 'pry'
end
```

Reply:
Done